reportportal/README.md

```diff
@@ -637,8 +637,8 @@ minio:
   enabled: true
   installdep:
     enable: false
-  endpoint: http://<minio-release-name>.default.svc.cluster.local:9000
-  endpointshort: <minio-release-name>.default.svc.cluster.local:9000
+  endpoint: http://<minio-release-name>-minio.default.svc.cluster.local:9000
+  endpointshort: <minio-release-name>-minio.default.svc.cluster.local:9000
```

Comment:
That's not always so, just in case release name is not `minio`. We have separate notes for postgres and rabbitmq explaining this. Please add it.

Reply:
postgres and rabbitmq use the patterns:
<rabbitmq-release-name>-rabbitmq.default.svc.cluster.local
<postgresql-release-name>-postgresql.default.svc.cluster.local
https://github.com/reportportal/kubernetes/blob/ed0ccef24a7d17175bff949b2fc86f681cf81485/reportportal/values.yaml#L186 & L198

It does not matter what release name is used. It's a some automatical suffix for endpoint.

Reply:
when I used the word 'minio' as release name the working endpoint was minio-minio.default.svc.cluster.local

Reply:
OK, but for me it's `minio.default.svc.cluster.local`. You might be using different chart, but we have to follow our recommended chart.

Reply:
Just checked again. Yep, you are right. If release name is equal to the chart name the endpoint contains only the chart name. And the same occurs for postgresql and rabbitmq endpoints.
But examples for psql, rmq and minio are different. And in my case I spend enough time to understand why it does not work. My idea is to standardize the examples.

Reply:
the helm commands are similar
https://github.com/reportportal/kubernetes/tree/develop/reportportal#7-minio-installation

but endpoint name patterns are different